src/renderer.js

```diff
@@ -64,6 +82,7 @@ function saveTask(task) {
 }
 
 function loadTasks() {
+  tasksList.innerHTML = "";
```

Comment:
Didn't really do anything, I tried adding this in hopes that it will fix the rendering bug but it didn't :(